plugins/couchbase/couchbaseembed.cpp

```diff
@@ -364,7 +383,7 @@ namespace couchbaseembed
         const char *user = "";
         const char *password = "";
         const char *bucketname = "default";
-        unsigned port = 8093;
+        unsigned port = 8091;
```

Comment:
Why is the port hardcoded, and can it be overridden?  Also consider adding to system\include\portlist.h